lambdas/db-migration/main.tf

```diff
@@ -70,7 +70,7 @@ resource "aws_lambda_function" "db_migration" {
   handler          = "index.handler"
   role             = aws_iam_role.db_migration.arn
   runtime          = "nodejs12.x"
-  timeout          = 120
+  timeout          = var.lambda_timeout
   memory_size      = 256
```

Comment:
Since we're changing defaults, maybe the memory size should get bumped up too, I just checked a few of the runs on my dev stack and ~they all~ many of them maxed out memory of 255 mb.

Reply:
I'm not sure that's *required*.   The log saying the container is using 256 doesn't mean it's going to run out of memory, it just means AWS allocated a bunch in the background and might be swapping.      

This lambda also does relatively little other than fire off migration queries.    

That said, there's no reason not to, generally increasing lambdas causes better cost efficiency (at increased risk with higher run times), but this shouldn't scale, so that's largely irrelevant.

Reply:
ab1ff38363